expression/builtin_regexp_test.go

```diff
@@ -948,6 +948,9 @@ func TestRegexpReplace(t *testing.T) {
 		{"abc abd abe", "ab.", "cz", "cz cz cz", "0x637A20637A20637A", nil},
 		{"你好 好的", "好", "逸", "你逸 逸的", "0xE4BDA0E980B820E980B8E79A84", nil},
 		{"", "^$", "123", "123", "0x313233", nil},
+		{"stackoverflow", "(.{5})(.*)", "\\\\+\\2+\\1+\\2+\\1\\", "\\+overflow+stack+overflow+stack", "0x5C2B6F766572666C6F772B737461636B2B6F766572666C6F772B737461636B", nil},
+		{"fooabcdefghij fooABCDEFGHIJ", "foo(.)(.)(.)(.)(.)(.)(.)(.)(.)(.)", "\\\\\\9\\\\\\8-\\7\\\\\\6-\\5\\\\\\4-\\3\\\\\\2-\\1\\\\", "\\i\\h-g\\f-e\\d-c\\b-a\\ \\I\\H-G\\F-E\\D-C\\B-A\\", "0x5C395C382D375C362D355C342D335C322D315C205C395C382D375C362D355C342D335C322D315C", nil},
```

Comment:
Using \` instead of " can make this string a raw string literal which can reduce `\`.

Reply:
> Using ` instead of " can make this string a raw string literal which can reduce `\`.

okk